service-front/config/application.config.php

```diff
@@ -2,19 +2,19 @@
 return array(
     // This should be an array of module namespaces used in the application.
     'modules' => array(
-        'Zend\Mvc\Console',
-        'Zend\Mail',
-        'Zend\Mvc\I18n',
-        'Zend\I18n',
-        'Zend\Mvc\Plugin\FlashMessenger',
-        'Zend\Form',
-        'Zend\InputFilter',
-        'Zend\Filter',
-        'Zend\Hydrator',
-        'Zend\Cache',
-        'Zend\Session',
-        'Zend\Router',
-        'Zend\Validator',
+        'Laminas\Mvc\Console',
+        'Laminas\Mail',
+        'Laminas\Mvc\I18n',
+        'Laminas\I18n',
+        'Laminas\Mvc\Plugin\FlashMessenger',
+        'Laminas\Form',
+        'Laminas\InputFilter',
+        'Laminas\Filter',
+        'Laminas\Hydrator',
+        'Laminas\Cache',
+        'Laminas\Session',
+        'Laminas\Router',
+        'Laminas\Validator',
         'Application',
         //ZfcTwig MUST be the last module!
         'ZfcTwig'
```

Comment:
is zfcTwig right here? is there a laminas replacement?

Reply:
Good Q but in fact the Twig plugin for Laminas is still called ZfcTwig.  In an ideal world I guess they would rename that too. Just for reassurance, Rich McHale told me Sirius after upgrade is using the exact same ZfcTwig version as we are moving to here.